mu-plugins/central-hub/tests/phpunit/unit/meta-data/saveMetaBoxes.php

```diff
@@ -0,0 +1,155 @@
+
+use Brain\Monkey;
+use function spiralWebDB\Metadata\save_meta_boxes;
+use function KnowTheCode\ConfigStore\loadConfig;
+use spiralWebDb\Cornerstone\Tests\Unit\Test_Case;
+
+/**
+ * Class Tests_SaveMetaBoxes
+ *
+ * @package spiralWebDb\centralHub\Tests\Unit\Metadata
+ * @group   meta-data
+ */
+class Tests_SaveMetaBoxes extends Test_Case {
+
+	/**
+	 * Prepares the test environment before each test.
+	 */
+	protected function setUp() {
+		parent::setUp();
+
+		require_once CENTRAL_HUB_ROOT_DIR . '/src/meta-data/meta-box.php';
+	}
+
+	/**
+	 * Test save_meta_boxes() should not save when no meta box keys are in the Config Store.
+	 */
+	public function test_should_not_save_when_no_meta_box_keys_are_in_store() {
+		Monkey\Functions\expect( 'spiralWebDB\Metadata\get_meta_box_keys' )
+			->once()
+			->withNoArgs()
+			->andReturn( [] );
+		Monkey\Functions\expect( 'spiralWebDB\Metadata\get_meta_box_id' )->never();
+		Monkey\Functions\expect( 'spiralWebDB\Metadata\is_okay_to_save_meta_box' )->never();
+		Monkey\Functions\expect( 'KnowTheCode\ConfigStore\getConfigParameter' )->never();
+		Monkey\Functions\expect( 'spiralWebDB\Metadata\save_custom_fields' )->never();
+		save_meta_boxes( 178 );
+	}
+
+
+	/**
+	 * Test save_meta_boxes() should not save when config meta box key does not begin with 'meta_box.'.
+	 */
+	public function test_should_not_save_when_config_meta_box_key_does_not_begin_with_metabox() {
+		Monkey\Functions\expect( 'spiralWebDB\Metadata\get_meta_box_keys' )
+			->once()
+			->withNoArgs()
+			->andReturn( [] );
+		Monkey\Functions\expect( 'spiralWebDB\Metadata\get_meta_box_id' )->never();
+		Monkey\Functions\expect( 'spiralWebDB\Metadata\is_okay_to_save_meta_box' )->never();
+		Monkey\Functions\expect( 'KnowTheCode\ConfigStore\getConfigParameter' )->never();
+		Monkey\Functions\expect( 'spiralWebDB\Metadata\save_custom_fields' )->never();
+		save_meta_boxes( 229 );
+	}
+
+	/**
+	 * Test save_meta_boxes() should not save when the $meta_box_key does not exist in $_POST.
+	 */
+	public function test_should_not_save_when_the_meta_box_key_does_not_exist_in_POST() {
+		Monkey\Functions\expect( 'spiralWebDB\Metadata\get_meta_box_keys' )
+			->once()
+			->withNoArgs()
+			->andReturn( [ 'meta_box.members' ] );
+		Monkey\Functions\expect( 'spiralWebDB\Metadata\get_meta_box_id' )
+			->once()
+			->with( 'meta_box.members' )
+			->andReturn( 'members' );
+		Monkey\Functions\expect( 'spiralWebDB\Metadata\is_okay_to_save_meta_box' )
+			->once()
+			->with( 'members' )
+			->andReturn( false );
+		Monkey\Functions\expect( 'KnowTheCode\ConfigStore\getConfigParameter' )->never();
+		Monkey\Functions\expect( 'spiralWebDB\Metadata\save_custom_fields' )->never();
+		save_meta_boxes( 317 );
+	}
+
+	/**
+	 *  Test save_meta_boxes() should not save when the nonce key is not equal to the expected value.
+	 */
+	public function test_should_not_save_when_the_nonce_key_is_not_equal_to_the_expected_value() {
```

Comment:
@rgadon107 This test is covered in the test below. If the value isn't expected, then the nonce check fails. Therefore, you can delete this test.